x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/index.tsx

```diff
@@ -0,0 +1,109 @@
+import { useInventoryAbortableAsync } from '../../hooks/use_inventory_abortable_async';
+import { useInventoryRouter } from '../../hooks/use_inventory_router';
+import { useKibana } from '../../hooks/use_kibana';
+import { GroupSelector } from './group_selector';
+import { InventoryGroupAccordion } from './inventory_group_accordion';
+import { groupCountCss } from './styles';
+import { useInventoryParams } from '../../hooks/use_inventory_params';
+
+export interface GroupedInventoryPageProps {
+  setGroupSelected: (selected: string) => void;
+}
+
+export function GroupedInventoryView() {
+  const { searchBarContentSubject$ } = useInventorySearchBarContext();
+  const {
+    services: { inventoryAPIClient },
+  } = useKibana();
+
+  const inventoryRoute = useInventoryRouter();
+  const { query } = useInventoryParams('/');
+  const { kuery, entityTypes } = query;
+
+  const { value = { groupBy: ENTITY_TYPE, groups: [] }, refresh } = useInventoryAbortableAsync(
+    ({ signal }) => {
+      return inventoryAPIClient.fetch('GET /internal/inventory/entities/group_by/{field}', {
+        params: {
+          path: {
+            field: ENTITY_TYPE,
+          },
+          query: {
+            kuery,
+            entityTypes: entityTypes?.length ? JSON.stringify(entityTypes) : undefined,
+          },
+        },
+        signal,
+      });
+    },
+    [entityTypes, inventoryAPIClient, kuery]
+  );
+
+  const totalEntities = value.groups.reduce((acc, group) => acc + group.count, 0);
+
+  useEffectOnce(() => {
+    const searchBarContentSubscription = searchBarContentSubject$.subscribe(
+      ({ refresh: isRefresh, ...queryParams }) => {
+        if (isRefresh) {
+          refresh();
+        } else {
+          inventoryRoute.push('/', {
+            path: {},
+            query: { ...query, ...queryParams },
+          });
+        }
+      }
+    );
+    return () => {
+      searchBarContentSubscription.unsubscribe();
+    };
+  });
+
+  return (
+    <>
+      <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween" alignItems="center">
+        <EuiFlexItem grow={false}>
+          <span css={groupCountCss}>
+            <FormattedMessage
+              id="xpack.inventory.groupedInventoryPage.entitiesTotalLabel"
+              defaultMessage="{total} Entities"
+              values={{ total: totalEntities }}
+            />
+          </span>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <span css={groupCountCss} style={{ borderRight: 'none' }}>
+            <FormattedMessage
+              id="xpack.inventory.groupedInventoryPage.groupsTotalLabel"
+              defaultMessage="{total} Groups"
+              values={{ total: value.groups.length }}
+            />
+          </span>
+        </EuiFlexItem>
+        <EuiFlexItem grow />
+        <EuiFlexItem grow={false}>
+          <GroupSelector />
+        </EuiFlexItem>
+      </EuiFlexGroup>
```

Comment:
I think this can be an extra component and be used by both the Unified and the Grouped views. You can mark the groupCount as options and skip that when on the Unified view. 

You also don't need this, do you:
```
<EuiFlexItem grow />
```